packages/@instana-integration/python/README.md

```diff
@@ -22,28 +22,45 @@ pip install opentelemetry-instrumentation-system-metrics
-| process.runtime.cpython.gc_count   | GC activity          | Number   |
-| process.runtime.cpython.memory | Memory usage              | Byte   |
-| system.disk.io       | I/O  | Number |
-| system.network.io       | Events  | Number |
+| Metrics Name                             | Description       | Unit       |
+|------------------------------------------|-------------------|------------|
+| process.runtime.cpython.context_switches | Context switching | Number     |
+| process.runtime.cpython.cpu.utilization  | CPU utilization   | Percentage |
+| process.runtime.cpython.thread_count     | Threads           | Number     |
+| process.runtime.cpython.cpu_time         | Time Spent        | S          |
+| process.runtime.cpython.gc_count         | GC activity       | Number     |
+| process.runtime.cpython.memory           | Memory usage      | Byte       |
+| system.disk.io                           | I/O               | Number     |
+| system.network.io                        | Events            | Number     |
 
 
 ### Resource Attributes
 
 Below are the resource attributes that are currently supported by this integration package.
 
-| Attribute Key              | Type |  Description           |
-|----------------------------|-------|------------------------|
-| service.name               | string  | This attribute is used to describe the entity name.    |
-| service.instance.id        | string  | This attribute is used to describe the entity ID of the current object.  |
+| Attribute Key              | Type   | Description                                                             |
+|----------------------------|--------|-------------------------------------------------------------------------|
+| service.name               | string | This attribute is used to describe the entity name.                     |
+| service.instance.id        | string | This attribute is used to describe the entity ID of the current object. |
 
-### Installation and Usage
+## Events
+
+Below are the events that are currently supported by this integration package.
+
+| Event Name                                        | Description       |
+|---------------------------------------------------|-------------------|
+| Excessive Context Switching in Python Application | —                 |
+| Frequent Garbage Collection in Python Application | —                 |
+| High CPU Utilization in Python Application        | —                 |
+| High Disk I/O in Python Application               | —                 |
+| High Memory Usage in Python Application           | —                 |
+| High Network I/O in Python Application            | —                 |
+| High Thread Count in Python Application           | —                 |
+| Memory Leak Detection in Python Application       | —                 |
+
+Note: The conditionValue is a reference value in event definitions. Please adjust this value based on your specific environment.
```

Comment:
What is conditionValue? If I'm a potential user, it may be confusing to me. Can it be more generic for the wording? Also, let's move it to the place right after the paragraph above the table, so that it's more easy to be found by the user.